README.md

```diff
@@ -55,7 +54,7 @@ The version format for this repo is `{major}.{minor}.{patch}` for stable, and
 
 To issue the next version in line, specify which part to bump,
 like `make release bump=minor` or `make release bump=devnum`. This is typically done from the
-master branch, except when releasing a beta (in which case the beta is released from master,
+main branch, except when releasing a beta (in which case the beta is released from main,
```

Comment:
The template and web3 are the only libs that have switched from `master` to `main` so far. The plan is to make the change at the same time as dropping py37, leading up to v7 web3 release.